meiki_server/auth/routes.go

```diff
@@ -74,7 +79,12 @@ func getCreateHandler(ctx context.Context, a Auth) gin.HandlerFunc {
 func getDeleteHandler(ctx context.Context, a Auth) gin.HandlerFunc {
 	return func(c *gin.Context) {
 		var creds Credentials
-		c.BindJSON(&creds)
+		err := c.BindJSON(&creds)
+
+		if err != nil {
```

Comment:
some drying opportunity here?

Can we add the parseCredential as a(nother) middleware instead? (thoughts for a different PR)

Reply:
Hmm, yep, will look into it

Reply:
Added as a TODO to be addressed in another PR